frontend/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

```diff
@@ -263,6 +262,9 @@ export function applyThemeDefaults(config: any, theme: Theme): any {
       continuousHeight: 350,
       continuousWidth: 400,
     },
+    mark: {
+      tooltip: true,
```

Comment:
nit: should we also use the colors from above? Or how do the tooltips currently look for non streamlit charts?

```json
mark: {
      tooltip: true,
      ...(hasLightBackgroundColor(theme)
        ? { color: "#0068C9" }
        : { color: "#83C9FF" }),
    },
```

Reply:
Looking the same so as a result, I don't think we need to copy this over.
<img width="816" alt="Screen Shot 2022-10-28 at 3 37 53 PM" src="https://user-images.githubusercontent.com/16749069/198746238-40dca35e-9ea1-4394-a96a-a4d8bf666811.png">
<img width="839" alt="Screen Shot 2022-10-28 at 3 37 56 PM" src="https://user-images.githubusercontent.com/16749069/198746239-512e1da2-9855-47de-8d10-b59eae81d33f.png">

Reply:
Oh, I see. I thought the colors where referring to the tooltips :)